src/jobs/models/jobManager.js

```diff
@@ -34,16 +38,35 @@ module.exports.reloadCronJobs = async () => {
     const configData = await configHandler.getConfig();
     try {
         const jobs = await databaseConnector.getJobs(contextId);
-        jobs.forEach(async function (job) {
+        for (const job of jobs) {
             if (job.cron_expression !== null) {
-                addCron(job, job.cron_expression, configData);
+                await addCron(job, job.cron_expression, configData);
             }
-        });
+        };
     } catch (error) {
         throw new Error('Unable to reload scheduled jobs, error: ' + error);
     }
 };
 
+module.exports.reloadChaosExperiments = async () => {
+    const contextId = httpContext.get(CONTEXT_ID);
+    const jobPlatform = await configHandler.getConfigValue(CONFIG.JOB_PLATFORM);
+    if (jobPlatform.toUpperCase() !== KUBERNETES) {
+        return;
+    }
+    try {
+        const timestamp = new Date().valueOf();
+        const futureJobExperiments = await chaosExperimentsManager.getFutureJobExperiments(timestamp, contextId);
+        for (const futureJobExperiment of futureJobExperiments) {
+            const calculatedStartAfter = futureJobExperiment.start_time - timestamp;
+            const chaosExperiment = await chaosExperimentsManager.getChaosExperimentById(futureJobExperiment.experiment_id);
+            jobExperimentHandler.scheduleChaosExperiment(chaosExperiment.kubeObject, futureJobExperiment.job_id, futureJobExperiment.id, calculatedStartAfter);
+        }
```

Comment:
I would try catch inside - if you failed bringing one - don't think you want to lose all the restarts

Reply:
done